Mathlib/CategoryTheory/Adjunction/Mates.lean

```diff
@@ -73,199 +74,524 @@ This can be seen as a bijection of the 2-cells:
 
 Note that if one of the transformations is an iso, it does not imply the other is an iso.
 -/
-def transferNatTrans : (G ⋙ L₂ ⟶ L₁ ⋙ H) ≃ (R₁ ⋙ G ⟶ H ⋙ R₂) where
-  toFun h :=
-    { app := fun X => adj₂.unit.app _ ≫ R₂.map (h.app _ ≫ H.map (adj₁.counit.app _))
-      naturality := fun X Y f => by
-        dsimp
-        rw [assoc, ← R₂.map_comp, assoc, ← H.map_comp, ← adj₁.counit_naturality, H.map_comp, ←
-          Functor.comp_map L₁, ← h.naturality_assoc]
-        simp }
-  invFun h :=
-    { app := fun X => L₂.map (G.map (adj₁.unit.app _) ≫ h.app _) ≫ adj₂.counit.app _
-      naturality := fun X Y f => by
-        dsimp
-        rw [← L₂.map_comp_assoc, ← G.map_comp_assoc, ← adj₁.unit_naturality, G.map_comp_assoc, ←
-          Functor.comp_map, h.naturality]
-        simp }
-  left_inv h := by
-    ext X
-    dsimp
-    simp only [L₂.map_comp, assoc, adj₂.counit_naturality, adj₂.left_triangle_components_assoc, ←
-      Functor.comp_map G L₂, h.naturality_assoc, Functor.comp_map L₁, ← H.map_comp,
-      adj₁.left_triangle_components]
-    dsimp
-    simp only [id_comp, ← Functor.comp_map, ← Functor.comp_obj, NatTrans.naturality_assoc]
-    simp only [Functor.comp_obj, Functor.comp_map, ← Functor.map_comp]
-    have : Prefunctor.map L₁.toPrefunctor (NatTrans.app adj₁.unit X) ≫
-        NatTrans.app adj₁.counit (Prefunctor.obj L₁.toPrefunctor X) = 𝟙 _ := by simp
-    simp [this]
-  -- See library note [dsimp, simp].
-  right_inv h := by
-    ext X
-    dsimp
-    simp [-Functor.comp_map, ← Functor.comp_map H, Functor.comp_map R₁, -NatTrans.naturality, ←
-      h.naturality, -Functor.map_comp, ← Functor.map_comp_assoc G, R₂.map_comp]
-#align category_theory.transfer_nat_trans CategoryTheory.transferNatTrans
-
-theorem transferNatTrans_counit (f : G ⋙ L₂ ⟶ L₁ ⋙ H) (Y : D) :
-    L₂.map ((transferNatTrans adj₁ adj₂ f).app _) ≫ adj₂.counit.app _ =
-      f.app _ ≫ H.map (adj₁.counit.app Y) := by
-  erw [Functor.map_comp]
-  simp
-#align category_theory.transfer_nat_trans_counit CategoryTheory.transferNatTrans_counit
-
-theorem unit_transferNatTrans (f : G ⋙ L₂ ⟶ L₁ ⋙ H) (X : C) :
-    G.map (adj₁.unit.app X) ≫ (transferNatTrans adj₁ adj₂ f).app _ =
-      adj₂.unit.app _ ≫ R₂.map (f.app _) := by
-  dsimp [transferNatTrans]
-  rw [← adj₂.unit_naturality_assoc, ← R₂.map_comp, ← Functor.comp_map G L₂, f.naturality_assoc,
-    Functor.comp_map, ← H.map_comp]
-  dsimp; simp
-#align category_theory.unit_transfer_nat_trans CategoryTheory.unit_transferNatTrans
-
--- See library note [dsimp, simp]
-end Square
+@[simps]
+def Mates :
+    (G ⋙ L₂ ⟶ L₁ ⋙ H) ≃ (R₁ ⋙ G ⟶ H ⋙ R₂) where
```

Comment:
Only names of types should be capitalised:
```suggestion
def mateEquiv : (G ⋙ L₂ ⟶ L₁ ⋙ H) ≃ (R₁ ⋙ G ⟶ H ⋙ R₂) where
```

Reply:
Ah whoops. I'll get to work on this.